client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button } from 'antd';
+import './style.css';
+
+const AdminCard = ({
+  imgUrl,
+  name,
+  description,
+  githbUrl,
+  student,
+  websiteLink,
+  studentNames,
+  internalProject,
+  remotelyProject,
+  editCard,
+  deleteCard,
+}) => {
+  return (
+    <div className="admin-card">
+      <div className="name-img-card">
+        <div className="img-card">
+          <img src={imgUrl} alt="card" />
+        </div>
+        <div>
+          <h3>Name</h3>
+          <p>{name}</p>
+        </div>
+      </div>
+      {description && (
+        <div>
+          <h3>Description</h3>
+          <p>{description}</p>
+        </div>
+      )}
+      <div>
+        <h3>Github link</h3>
+        <a href={githbUrl}>Click</a>
```

Comment:
you will send the user to another page so don't let it leave your website
so 
```suggestion
        <a href={githbUrl} target="blank" >Click</a>
```